packages/cozy-harvest-lib/src/stories/AccountSelectorHeader.fixtures.ts

```diff
@@ -0,0 +1,341 @@
+export const AccountSelectorHeaderFixtures = {
```

Comment:
this should be renamed somehow, these data arent linked to the header but to a whole konnector modal

Reply:
Also maybe there is still some personal data in there that should be removed